conversation/conversation.go

```diff
@@ -0,0 +1,70 @@
+package conversation
+
+import (
+	"net/http"
+
+	messagebird "github.com/messagebird/go-rest-api"
+)
+
+// ListOptions can be used to set pagination options in List().
+type ListOptions struct {
+	Limit, Offset int
+}
+
+// StartRequest contains the request data for the Start endpoint.
+type StartRequest struct {
+	ChannelID string          `json:"channelId"`
+	Content   *MessageContent `json:"content"`
+	To        string          `json:"to"`
+	Type      MessageType     `json:"type"`
+}
+
+// UpdateRequest contains the request data for the Update endpoint.
+type UpdateRequest struct {
+	Status ConversationStatus `json:"status"`
+}
+
+// DefaultListOptions provide a reasonable default for paginated endpoints.
+var DefaultListOptions = &ListOptions{10, 0}
+
+// List gets a collection of Conversations. Pagination can be set in options.
+func List(c *messagebird.Client, options *ListOptions) (*ConversationList, error) {
+	convList := &ConversationList{}
+	if err := request(c, convList, http.MethodGet, path, options); err != nil {
```

Comment:
Self review: add query parameters to the path. The value for `data` is sent as a JSON encoded body, which doesn't work with GET.

Reply:
Resolved in https://github.com/messagebird/go-rest-api/pull/49/commits/430d19fb45c9cf0ee346a0d76a3b611ca78cf2be